<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartDemo</class>
 <widget class="QWidget" name="ChartDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChartDemo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="themelabel">
       <property name="text">
        <string>主题</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="themeBox"/>
     </item>
     <item>
      <widget class="QLabel" name="donghua">
       <property name="text">
        <string>动画</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="donghuaBox"/>
     </item>
     <item>
      <widget class="QLabel" name="tuli">
       <property name="text">
        <string>图例</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tuliBox"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>抗锯齿</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>332</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QChartView" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
